### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by sumancrest0001**

### CODE REVIEW

**Comment by mosaaleb** [comment]
Reviewing

File: index.html [hunk 1]
```diff
@@ -1,42 +1,95 @@
...
-  </div>
+    <div class="wrapper">
+        <div class="header">
+            <div>
+                <nav class="top-nav">
+                    <ul>
+                        <li class="nav-items logo" id="apple-icon"><a href="#"><img src="https://assets.webiconspng.com/uploads/2017/09/Apple-Logo-PNG-Image-88327.png"></a></li>
+                        <span></span>
+                        <li class="nav-items store"><a href="#">Store</a></li>
+                        <span></span>
```
Comment (mosaaleb) [inline index.html:23]: There is no written styles for these classes in the css file `nav-items logo store mac ipod ...`. We can remove them if they don't do anything.

File: index.html [hunk 2]
```diff
@@ -1,42 +1,95 @@
...
+                        <li class="nav-items ipad"><a href="#">iPad</a></li>
+                        <span></span>
+                        <li class="nav-items itunes"><a href="#">iTunes</a></li>
+                        <span></span>
+                        <li class="nav-items support"><a href="#">Support</a></li>
+                        <span></span>
+                        <li class="search-field">
+                            <div id="search-form">
+                                <form>
+                                    <i class="fa fa-search"></i><input type="text" name="search">
```
Comment (mosaaleb) [inline index.html]: For simple UI components like [search field](https://stackoverflow.com/questions/3294572/is-input-well-formed-without-a-form), there is no need to add `<form>` tag. `<input>` tag would be enough.

File: index.html [hunk 3]
```diff
@@ -1,42 +1,95 @@
...
+                                </form>
+                            </div>
+                        </li>
+                    </ul>
+                </nav>
+                </div>
+                <!-- title contents -->
+            <div class="content-section">
+                <p><a href="#">What will your verse be?</a></p>
+            </div>
```
Comment (mosaaleb) [inline index.html]: Main content links (the three text):
- [ ] Each link is an `<a>` HTML tag, with a `<div>` tag inside it.
- [ ] The `<div>` tag  inside the `<a>` tag should use an image as background -- [ ] The text on the original page is actually an image

File: index.html [hunk 4]
```diff
@@ -1,42 +1,95 @@
...
+            <div class="content-3">
+                <p><a href="#"> Watch the "Your Verse" film and explore the stories.</a><span><i class="fa fa-play-circle"></i></span></p>
+            </div>
+            
+        
+    </div>
+        <div class="middle-section">
+            <div class="articles article-1"><a href="#"><img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5s.jpg" alt="first article"></a></div>
+            <div class="articles article-2"><a href="#"><img src="images/second-article.jpg" alt="second article"></a></div>
```
Comment (mosaaleb) [inline index.html]: The four promo images part:
- [ ] Is made of four `<a>` tags with `<div>` tags inside it
- [ ] The `<div>`s inside the `<a>` tags should use an image as a background, [why `bacground-image` and not `` tag](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image)

**Review by mosaaleb — CHANGES_REQUESTED** [review]
### Hi @sumancrest0001 :wave:

#### Good job :clap:. You have met most of the requirements, yet there is minor requirements we need to work on before moving on.

Please see my comments below.

**Also** Let's add [stickler](https://github.com/microverseinc/linters-config/tree/master/css) to make sure that our code have good style guidelines.

Please handle these requirements and submit a new code review.

#### Happy Coding :computer:

**Comment by Torres-ssf** [comment]
### Nicely done @sumancrest0001 :+1: 

I notice that you fixed the stickler file in a separate branch but would be better if you merged to this current branch instead of merging to master since it needs to look for errors in the project before merging to master. The silver lining here is that you already know how to set up the linter :muscle: . We need to fix that. Please make the linter work in this branch so we can see if your code has no errors. I know that in the past we didn't pay much attention to it but linting is important to reduce errors and improve the overall quality of your code. 


- For Future Projects

- [ ] You should use Github/Git Flow in future projects. It's really easy to use it and will make your repository looks more professional. You will find instructions in the Patwright lesson description.

Kindly make this change and submit for code review again.

**Review by mosaaleb — APPROVED** [review]
## Amazing work @sumancrest0001 :clap:

You can merge into master :+1: 

While all the checks gives warning and not failures. Let's make sure in the coming project we handle all the warning also. 

Let's make sure from the next project that we follow the github/git flow instructions in your pathwright.

Good luck with the next projects

#### Happy Coding :computer:

**Pull request merged**